frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-form/upload-pvc-form.tsx

```diff
@@ -574,6 +561,26 @@ export const UploadPVCPage: React.FC<UploadPVCPageProps> = (props) => {
             inProgress={!loadedTemplates || !loadedPvcs || isCheckingCertificate}
             errorMessage={errorMessage}
           >
+            {isFileRejected ? (
+              <Alert variant="warning" isInline title="File type extension">
+                <p>
+                  Based on the file extension it seems like you are trying to upload a file which is
+                  not supported (
+                  {fileNameExtension
+                    ? `detected file extension is '${fileNameExtension}'`
+                    : 'no file extention detected'}
+                  ).
+                </p>
+                <p>
+                  <ExternalLink
+                    text="Learn more about supported formats"
+                    href={CDI_UPLOAD_SUPPORTED_TYPES_URL}
+                  />
+                </p>
+              </Alert>
+            ) : (
+              undefined
```

Comment:
why ternary condition?
just do `isFileRejected &&`

Reply:
`isFileRejected` is boolean, so if we use `&&` we get into a grey area of how to render `false` [1] , I preferred here to be explicit and make sure I understand `isFileRejected` is boolean, by explicitly render `{undefined}` instead of `false` which is redundant but explicit. On the other hand it does looks prettier with `&&` so changing to using `&&` :+1: 

[1] https://react-cn.github.io/react/tips/false-in-jsx.html